tests/ui/impl-trait/in-trait/auxiliary/rpitit.rs

```diff
@@ -1,3 +1,6 @@
+// [next] compile-flags: -Zlower-impl-trait-in-trait-to-assoc-ty
+// revisions: current next
```

Comment:
ok I looked into it -- we don't need `revisions` here. We "inherit" the revisions from the parent. But it *does* inherit the revisions from the parent, so no extra file needed like in my other PR. Let me fix that.

```suggestion
```

Reply:
we probably want the `compile-flags` anyway, I'd say.

Reply:
In any case, feel free to close this PR and leave yours open that aggregates this stuff.

Reply:
Yes, the compile-flags is necessary to enable the `-Z` flag when `[next]`, but re-declaring "`// revisions`" here does nothing.